fzf-tab.zsh

```diff
@@ -418,7 +418,8 @@ fzf-tab-complete() {
         } always {
             IN_FZF_TAB=0
         }
-        zle redisplay
+        zle .reset-prompt
```

Comment:
`reset-prompt` will cause a noticeable refresh, I suggest use it only when `_fzf_tab_continue` is true.

Reply:
Done.